src/utils/config_loader.ts

```diff
@@ -0,0 +1,10 @@
+import { get } from 'lodash';
+import { CustomConfig, WidgetConfig } from '../interfaces/custom_config';
+
+export class ConfigLoader {
+	constructor(private _config: CustomConfig) {}
+
+	public getInputDefinition(): WidgetConfig {
```

Comment:
That's not necessary. You can just traverse all the properties in `widget`, instead of creating a method to get each one.